### ISSUE TITLE
[REC] Light's Judgment racial never recommended on Demonology

### ISSUE THREAD

**Issue by dubudevs** [Hekili/hekili#3774]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Warlock - Demonology

### Describe the Issue

Lightforged draenei racial does a lot of damage in aoe but it is never recommended. For Demonology it should be used outside tyrant ramp on X+ targets, ideally configurable but otherwise just 4+. Asked in the demo APL discussion and they said just send it outside tyrant on cd. I dont think they want to bother theorycrafting for such a low play rate suboptimal race.

Its definitely worth using for other specs and classes in aoe but I do not know when specifically.

### How to Reproduce

Be lightforged draenei
Do dam

### Snapshot (Link)

https://pastebin.com/j4jm9RHt

**Comment by syrifgit** [comment]
The addon and also the sims APL indicate it should be used during tyrant. Why do you think that's wrong?

**Comment by Hekili** [comment]
I don't see any Warlock sim profiles that recommend Light's Judgment.  Is there a custom sim that reflects a DPS gain in multi-target scenarios?  Syrif is right that other racials are only recommended during Tyrant (or at the end of a fight), but the included racials are only buffs.

**Comment by dubudevs** [comment]
On a 5 min 10 boss sim it doesnt really change overall dps because its a long cd
https://www.raidbots.com/simbot/report/oeedNttdpdQq1QreYBHnpy/simc 
https://www.raidbots.com/simbot/report/79Uo5cSqEgEk8B9R9KXQRe/simc

On a 1 min sim it does increase it a little. 
https://www.raidbots.com/simbot/report/17nq7ABPinf48n5KYWYC7m
https://www.raidbots.com/simbot/report/fP17CetLMHYN1NoAZXnwPK

On 4 boss 1 min its also slightly higher
https://www.raidbots.com/simbot/report/dAJRX5d1bJhHyU1ypL2DWf
https://www.raidbots.com/simbot/report/rQEx1EnExD6261ViLJ1N23

Of course its not insane, but personally I'd like it if it was recommended since its definitely not a loss in aoe. Not sure if theres better optimisation than what I did, probably, but I doubt anyone will figure it out.

**Comment by Hekili** [comment]
There are a couple of entries for it on Raidbots that I'm not seeing when my script builds the priority. I'll look into it.

**Comment by dubudevs** [comment]
> There are a couple of entries for it on Raidbots that I'm not seeing when my script builds the priority. I'll look into it.

I added those 2 lines myself to compare. The default APL does not use it.

**Comment by Hekili** [comment]
Ah, got it.

**Comment by Hekili** [comment]
Are you willing to do a little additional research in terms of timing the usage of Light's Judgment?  I have a feeling we want to avoid having it come up during Tyrant/ramping so that shard generation isn't impacted at a less-than-ideal time.

**Comment by dubudevs** [comment]
> Are you willing to do a little additional research in terms of timing the usage of Light's Judgment? I have a feeling we want to avoid having it come up during Tyrant/ramping so that shard generation isn't impacted at a less-than-ideal time.

The lines I added are basically just precombat (since its a 3s delay) however a pet cancel attack macro is needed on pet classes as they will attack by default when its used, but people (the 2 people using this class/race combo and hekili) can use their brains and only use it when its suitable. 

The second line I added just casts it any time tyrant isnt up and the cd is at least 15s, to avoid ramp and tyrant windows. In theory this timing doesnt leave it on cd for more than ~15s ever, but its not critical to keep it on cd all the time. It's mostly just so its recommended at all.

And of course it should only ever be recommended in aoe.
